config/repo-values.yaml

```diff
@@ -83,6 +83,8 @@ sync:
       url: https://charts.yourls.org
     - name: hazelcast
       url: https://hazelcast.github.io/charts/
+    - name: hazelcast-jet
+      url: https://hazelcast.github.io/charts/
```

Comment:
Both the `hazelcast` and `hazelcast-jet` repositories share the same repository URL. All of the charts listed in a repository are shown on the Helm hub. Having the same repository URL twice will not add any new charts.

Did you mean to have a second Helm repository?

Reply:
You are right, we do not need this PR to list `hazelcast-jet charts`. I have created this PR because as I mentioned above `hazelcast/hazelcast-jet` and `hazelcast/hazeclast-jet-enterprise` charts were not listed when I search with `hazelcast` keyword. I have checked Helm Hub again, I could list all related chart properly:
https://hub.helm.sh/charts/hazelcast
<img width="1430" alt="Screen Shot 2020-06-02 at 09 13 42" src="https://user-images.githubusercontent.com/6005622/83486192-62160b80-a4b1-11ea-9af7-362e1d86cc12.png">
So we can close the PR. Thanks for your help 👍